CHANGELOG.md

```diff
@@ -23,6 +23,11 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/aws-infrastructure/compare/release-3.1.1...master
 
+### Added
+- Add `DataCenterFormula.Builder.jiraSharedStorageConfig` function to configure DC tests to store attachments/avatars in an S3 bucket.
+- Add `JiraSharedStorageConfig`.
+- Update AWSCli to support V2 and use V2.9.12 by default.
```

Comment:
When 2 CLI versions dropped, this could become an implementation detail that could be skipped in changelog